src/native/libs/System.Security.Cryptography.Native/apibridge_30.h

```diff
@@ -16,3 +27,5 @@ int local_EVP_PKEY_CTX_set_rsa_oaep_md(EVP_PKEY_CTX* ctx, const EVP_MD* md);
 int local_EVP_PKEY_CTX_set_rsa_padding(EVP_PKEY_CTX* ctx, int pad_mode);
 int local_EVP_PKEY_CTX_set_rsa_pss_saltlen(EVP_PKEY_CTX* ctx, int saltlen);
 int local_EVP_PKEY_CTX_set_signature_md(EVP_PKEY_CTX* ctx, const EVP_MD* md);
+
+int CRYPTO_set_mem_functions11(CRYPTO_malloc_fn malloc_fn, CRYPTO_realloc_fn realloc_fn, CRYPTO_free_fn free_fn);
```

Comment:
Why are you defining a variant name of CRYPTO_set_mem_functions here?  If it's the same signature since 1.1 it seems like it should be in `apibridge.h` (OSSL 1.1 API to use when building against OSSL 1.0 headers), and then it wouldn't need the suffix